Samples/Swift/DaysUntilBirthday/Shared/ViewModels/AuthenticationViewModel.swift

```diff
@@ -22,6 +22,13 @@ final class AuthenticationViewModel: ObservableObject {
   /// The user's log in status.
   /// - note: This will publish updates when its value changes.
   @Published var state: State
+
+#if os(iOS)
+  /// The user's account verification status.
+  /// - note: This will publish updates when its value changes.
+  @Published var verificationState: VerificationState
```

Comment:
I think it might be cleaner to create a `VerifiedAgeViewModel` that will hold this state. This will keep the `AuthenticationViewModel` tightly coupled to its view, which will do the same for `VerificationView`. Otherwise, `AuthenticationViewModel` will have some stuff that is only relevant to separate views, making it an "uber" view model.

Reply:
Makes sense, went ahead and created the new view model.